code/ecdsap256/Hacl.P256.fsti

```diff
@@ -114,7 +138,19 @@ val ecdsa_sign_p256_sha512: result: lbuffer uint8 (size 64)
   )
 
 
-[@ (Comment " Input: result buffer: uint8[64], \n m buffer: uint8 [mLen], \n priv(ate)Key: uint8[32], \n k (nonce): uint32[32]. 
+[@ (Comment "P256 signature WITHOUT hashing.
+
+This is NOT RECOMMENDED. Please use of the combined hash-and-sign functions
```

Comment:
It's not about it. It's more that we expect to have a hash here. This is the function used by Tezos iirc.

Reply:
I rewrote.